domain/auth/auth.go

```diff
@@ -96,7 +97,9 @@ func CtxWithAccessToken(ctx context.Context, at AccessToken) context.Context {
 // read (GET) the resource at the /ping path. This is obviously
```

Comment:
Let's change the struct comment to 

// CasbinEnforcer holds the casbin.Enforcer struct